acceptance-tests/src/main/kotlin/org/ostelco/at/jersey/Tests.kt

```diff
@@ -1,7 +1,6 @@
 package org.ostelco.at.jersey
 
 import org.junit.Test
-import org.ostelco.at.common.Firebase
```

Comment:
Did you remove the Firebase dependency from gradle settings

Reply:
Thanks, I will fix this